sections/appendix/models.tex

```diff
@@ -0,0 +1,57 @@
+\subsection{Extensions to our models}\label{subsec:app_exten_models}
+Looking at the results in \autoref{tab:metric_results}, the Author-Topic and Category-Topic model do not get very high scores in Topic Coherence.
+Trying to improve these results we have run the combination model using the same setup as the Author-Category model, which is explained in \autoref{subsec:combinations}, just exchanging one feature with the document feature.

```

Comment:
er ikke superglad for denne beskrivelse af author-doc.
specifikt det med 'just exchanging one feature with the document feature', syntes jeg ikke er særlig forklarende.
```suggestion
In attempt to improve these results, we combine the original \gls{lda} model with these models, adding a topic distribution for each document and combining it with the existing topics distributions, as with the Author-Category model, which is explained in \autoref{subsec:combinations}.
```